### DISCUSSION TITLE
Show time in the new routing mode based on speed

### DISCUSSION THREAD

**Discussion by MxtAppz** [organicmaps/organicmaps#6172] [Ideas]
Hello,

My idea is that it would be nice if the new routing mode (I don't know how is it called, but I mean the one which shows a straight line from the origin to the destination) had an option to allow the user entering a constant speed (either in kph or in mph) and calculate the time it would take flying at this speed for the whole route to arrive there. What do you think?

Thanks in advanced.

**Comment by biodranik** [comment]
The idea is interesting, but it should be easy to type/specify for anyone. What is your use-case for it?

**Reply by MxtAppz** [reply]
My idea is to place a set speed button in the bar where the distance label is but on the other side, which brings up a dialog asking for the speed and next to it a unit selector, and once set, to place the time on the same bar as the distance (but allowing the user to change speed again).

**Reply by biodranik** [reply]
That's an interesting idea. We discussed that the bottom bar segments distances can be removed when we can draw distances right on the map segments. So only the total distance will be visible, together with the custom speed and time (at the center).
CC @strump
